### ISSUE TITLE
Winter theme: Sidebar menu hover have no effect

### ISSUE THREAD

**Issue by sukhwinder33445** [Icinga/icingaweb2#4708]
## Describe the bug
There is no hover effect for sub nav items.
Keyboard navigation is not visible.

![Screenshot 2022-03-22 at 13 53 02](https://user-images.githubusercontent.com/54990055/159495880-6f0798bf-628b-4745-8bd4-5da0e4a28b3f.png)

## To Reproduce

1. Select any main menu in sidebar.
2. Move mouse pointer to any sub nav item.

There is no visible hover effect.
